ta/os_test/sub.mk

```diff
@@ -8,7 +10,7 @@ srcs-y += init.c
 srcs-y += os_test.c
 srcs-y += ta_entry.c
 srcs-$(CFG_TA_FLOAT_SUPPORT) += test_float_subj.c
-ifneq ($(COMPILER),clang)
+ifeq ($(WITH_CXX_TESTS),1)
```

Comment:
s/1/y/ ?

Reply:
Good catch, fixed and tested properly with both `WITH_CXX_TESTS` set to `y` and `n` this time.